components/BottomAppBar/src/MDCBottomAppBarView.m

```diff
@@ -25,6 +25,7 @@
 static NSString *kMDCBottomAppBarViewAnimKeyString = @"AnimKey";
 static NSString *kMDCBottomAppBarViewPathString = @"path";
 static NSString *kMDCBottomAppBarViewPositionString = @"position";
+static const CGFloat kMDCBottomAppBarViewFloatingButtonCenterToNavigationBarTopOffset = 0.f;
 static const CGFloat kMDCBottomAppBarViewFloatingButtonElevationPrimary = 6.f;
 static const CGFloat kMDCBottomAppBarViewFloatingButtonElevationSecondary = 4.f;
 static const int kMDCButtonAnimationDuration = 200;
```

Comment:
Filed https://github.com/material-components/material-components-ios/issues/5177 to clean up these constants.